_data/compatibility/samourai.yaml

```diff
@@ -0,0 +1,97 @@
+      shows_replaced_version: "true"
+      shows_original_version: "false"
+  examples:
+    - image: /img/compatibility/samourai/settings-rbf.png
+      caption: >
+        Sending RBF Transaction - Settings for sending RBF enabled transactions. RBF disabled by default.
+    - image: /img/compatibility/samourai/send-screen-default.png
+      caption: >
+        Sending RBF Transaction - Default send transaction screen. No RBF options.
+    - image: /img/compatibility/samourai/send-stonewall-prompt.png
+      caption: >
+        Sending RBF Transaction - Prompt during send transaction for “STONEWALL” feature.
+    - image: /img/compatibility/samourai/transaction-list-sent.png
+      caption: >
+        Sending RBF Transaction - Sent RBF enabled transaction. No RBF flag.
+    - image: /img/compatibility/samourai/transaction-details-sent.png
+      caption: >
+        Bumping RBF Enabled Transaction - Transaction details for RBF enabled transaction with Increase TX Fee option. Interesting note here - If there are no additional funds to pay for the bump, Samourai just fails silently here with no message and takes the user back to the transaction list screen.
+    - image: /img/compatibility/samourai/sent-transaction-increase-fee.png
+      caption: >
+        Sending RBF Transaction - Prompt when “increase tx fee” is chosen. Miner fee not customizable.
+    - image: /img/compatibility/samourai/transaction-list-sent-replaced.png
+      caption: >
+        Sending RBF Transaction - Send RBF bumped transaction replaces original RBF enabled transaction. No flag. Confirmation message does say “RBF transaction sent”
+    - image: /img/compatibility/samourai/transaction-list-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Transaction list screen for receiving RBF enabled transaction. No RBF flag.
+    - image: /img/compatibility/samourai/transaction-details-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Transaction details prompt for received transaction. “View transaction” defaults to Smartbit explorer (explorer configurable in settings).
+    - image: /img/compatibility/samourai/transaction-details-incoming-rbf-increase-fee.png
+      caption: >
+        Receiving RBF Transaction - Transaction details “Increase transaction fee” dialog.
+    - image: /img/compatibility/samourai/transaction-list-incoming-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - Transaction list screen showing both original and bumped transactions. Total reflecting sum of both transactions. No RBF flag or warnings.
+    - image: /img/compatibility/samourai/transaction-list-replacement-confirmed.png
+      caption: >
+        Receiving Bumped RBF Transaction - Transaction list screen updated to just show replacement transaction after a period of time. Even before either RBF transaction was confirmed.
+segwit:
+  tested:
+    date: "2019-07-25"
+    platforms:
+      - Android
+    version: "0.99.82"
```

Comment:
I don't understand this versioning. Version tested for RBF was 3.3.2, and this is 0.99.82?

Reply:
This version is wrong. It is 0.81. Good catch.